README.md

```diff
@@ -56,6 +56,22 @@ doc = DocumentFile.from_pdf("path/to/your/doc.pdf").as_images()
 result = model(doc)
 ```
 
+### Deal with the rotation issues
+You can provide rotated documents, and/or documents with different boxes orientations.
+We provide different options to deal with such cases:
```

Comment:
Should you use docTR on documents that include rotated pages, or pages with multiple box orientations, you have multiple options to handle it: